client/utils/tags.tsx

```diff
@@ -69,7 +69,7 @@ export function injectConditionalTags(
   ) as psl.ParsedDomain;
 
   // inject gap all production languages except Chinese
-  if (parsedHomeUrl.subdomain === 'www' && parsedHomeUrl.tld === 'org') {
+  if (env.clientLocale !== 'chinese' && parsedHomeUrl.tld === 'org') {
```

Comment:
We could do a similar thing here, rather than rely on `env.clientLocale`, which will always be `'english'` in the CI like @naomi-lgbt mentioned:

```suggestion
  if (homeLocation.split('/').pop() !== 'chinese' && parsedHomeUrl.tld === 'org') {
```